tests/test_wave_01.py

```diff
@@ -181,14 +171,12 @@ def test_moves_movie_from_watchlist_to_watched():
 
     # Assert
     assert len(updated_data["watchlist"]) == 1
-    assert len(updated_data["watched"]) == 2
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert len(updated_data["watched"]) == 2 #this indicates two movies watched
+    # ****** Add assertions here to test that the correct movie was added to "watched" ********** AT
+    assert updated_data["watched"][0] == FANTASY_2 #we need to assert both expected movies 
+    assert updated_data["watched"][1]== HORROR_1 #the function returns two movies watched 
```

Comment:
These asserts indicate that you know what the `updated_data["watched"]` list should be. Rather than check each index individually, they could be combined into a single assert that makes sure the list looks exactly as expected. Something like the following:

 ```suggestion
    assert updated_data["watched"] == [FANTASY_2, HORROR_1] 
    
```

As mentioned before, it might be a good idea to remove the comments on these asserts as well!